frontend/src/Account.js

```diff
@@ -15,57 +17,120 @@ function Account() {
                 if (accounts.length > 0) {
                     setAuthenticated(true);
                     setAccount(accounts[0]);
+                    console.log(accounts[0]);
                 }
                 else {
-                    navigate("/login")
+                    alert ("There was an error, please try again");
                 }
             });
     }, []);
 
-    // returns all nfts listed by an account, to be checked later
+    // returns all nfts for the logged in account
     useEffect(() => {
-        if (isAuthenticated && account !== undefined) {
```

Comment:
What happens if account is undefined

Reply:
Added a condition to check that account is defined